README.md

```diff
@@ -102,6 +102,13 @@ Sample setting:
 ```
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
+## Developer's Guide
+
```

Comment:
At the moment CI does `yarn && yarn compile`
https://github.com/eisopux/checker-framework-vscode/blob/master/azure-pipelines.yml#L31

Is that needed in addition to `vsce package` or should CI be changed?

Reply:
I'll have to reinstall the extension, I did the steps I described here, but it is very likely that this CI does it already, in which case this is just extra information